mlrun/artifacts/model.py

```diff
@@ -406,7 +467,9 @@ def _get_file_body(self):
         return mlrun.get_dataitem(target_model_path).get()
 
 
-def get_model(model_dir, suffix=""):
+def get_model(
+    model_dir: Optional[str] = None, suffix="", artifact: Optional[ModelArtifact] = None
```

Comment:
Maybe you can this, and it will simplify your code
 ```suggestion
    model_dir:Union[str, ModelArtifact] = None, suffix="",
```